src/plugins/intel_cpu/tests/functional/custom/subgraph_tests/src/index_add_scatter_elements_update.cpp

```diff
@@ -0,0 +1,293 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "openvino/opsets/opset13.hpp"
+#include "openvino/pass/manager.hpp"
+
+#include "shared_test_classes/base/ov_subgraph.hpp"
+#include "utils/cpu_test_utils.hpp"
+#include "common_test_utils/include/common_test_utils/ov_tensor_utils.hpp"
+
+using namespace ov::test;
+using namespace CPUTestUtils;
+using namespace ov::op;
+
+namespace ov {
+namespace test {
+
+/*
+  This test runs a graph that is equivelent to torch.Tensor.index_add_.
```

Comment:
Could you please insert the subgraph scheme?

Reply:
The subgraph is snipped from torch frontend which maps index_add_ to SEU-12. The unit test is not mandatory to this PR. I create it to visualize the pattern and debugging and profiling purpose.